extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,251 @@
+  "title": "Post Attributed Conversion Events",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Post Attributed Conversion Events. Conversion Path can contain at the most 10 events that contributed to the conversion event. ",
+  "definitions": {
+    "attributedconversions": {
+      "properties": {
+        "xdm:triggerType": {
+          "title": "Trigger Event Type",
+          "type": "string",
+          "description:": "Trigger Type for an Event - examples include click, impression"
+        },
+        "xdm:pathPosition": {
+          "title": "Path Position",
+          "type": "string",
+          "description": "Path Position to conversion"
+        },
+        "xdm:logId": {
+          "title": "Internal identifier for the batch",
+          "type": "number",
+          "description": "Internal identifier for the batch"
+        },
+        "xdm:conversionTime": {
+          "title": "Conversion Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Conversion Time of the Event"
+        },
+        "xdm:triggerTime": {
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Trigger Time of the Event"
+        },
+        "xdm:originaltriggerTime": {
+          "title": "Original Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Original Trigger Time"
+        },
+        "xdm:advertiserId": {
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Advertiser Identifier"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name."
+        },
+        "xdm:displaycampaignId": {
+          "title": "Display Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the display advertising platform."
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the search advertising platform."
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform."
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform."
+        },
+        "xdm:displayadId": {
+          "title": "Display Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the display advertising platform."
+        },
+        "xdm:adId": {
```

Comment:
change field id to searchAdId? would there be a situation where both searchAdId and displayAdId present with a conversion? If not, can we merge them to adId?